app/Database/database.sql

```diff
@@ -9,14 +9,14 @@ CREATE TABLE Station (
 	id_station INTEGER PRIMARY KEY,
     max_capacity INTEGER
 );
-CREATE TABLE Locomotora (
-	id_locomotora INTEGER,
+CREATE TABLE Locomotive (
```

Comment:
Me parece todo bien, pero creo que el concepto más adecuado es Travel no Journey.
e.g:

travel
Air travel is faster than rail travel.
His job involves a lot of travelling.

journey
We had a long journey through the mountains.
He usually reads the paper during the train journey to work.

Reply:
Para ser sincero la palabra "Travel" va con el contexto y es mas cómoda para ocuparla, lo que sí, tome como referencia la momento de seleccionar la palabra Travel = viaje ida y vuelta: journey = vieje de solo ida.
Pero podemos ver cual nos acomoda más.